src/Common/src/Interop/Windows/WebSocket/Interop.WebSocketAbortHandle.cs

```diff
@@ -10,7 +10,6 @@ internal static partial class Interop
     internal static partial class WebSocket
     {
         [DllImport(Libraries.WebSocket)]
-        [SuppressUnmanagedCodeSecurity]
```

Comment:
We want to leave all attributes in all Common code? Or just those used in OOBs (which was my goal)?